themes/default/content/docs/using-pulumi/continuous-delivery/gitlab-ci.md

```diff
@@ -22,83 +22,82 @@ aliases:
 staging and production stacks based on commits to specific Git branches. This is sometimes
 referred to as Push-to-Deploy.
 
-Pulumi doesn't require any particular arrangement of stacks or workflow to work in a
-continuous integration / continuous deployment system. So the steps described here can be
-altered to fit into any existing type of deployment setup.
+Using GitLab with Pulumi is a great way to combine software development and infrastructure
+as code (IaC) best practices. Pulumi is built with CI/CD integration in mind because:
+
+- It doesn't require any particular arrangement of stacks or workflow to work in CI/CD processes.
+- It can be run from anywhere and your infrastructure code can be hosted anywhere.
+
+The examples provided in this guide were created under the context of a Pulumi TypeScript project with the project code hosted in a GitLab repository,
+but the steps described below can be altered to fit into any existing type of deployment setup.
 
 ## Prerequisites
 
-- An account on [https://app.pulumi.com](https://app.pulumi.com) and that you have created a new project.
-    - This just means you will sign-in using your GitLab credentials.
-    - However, pulumi can be run from anywhere and your infrastructure code itself can be hosted anywhere.
-- The [latest CLI](/docs/install/).
-- A bare repo and set the remote URL to be your GitLab project.
+- Sign up for a [Pulumi account](https://app.pulumi.com) using your GitLab credentials.
```

Comment:
Is it necessary to use your GitLab credentials for the Pulumi Cloud account? Doesn't seem like it should be.